src/panels/config/area_registry/ha-config-area-registry.ts

```diff
@@ -48,19 +50,23 @@ class HaConfigAreaRegistry extends LitElement {
       `;
     }
     return html`
-      <hass-subpage header="Area Registry">
+      <hass-subpage
+        header="${this.hass.localize("ui.panel.config.area_registry.caption")}"
+      >
         <ha-config-section .isWide=${this.isWide}>
           <span slot="header">
             ${this.hass.localize("ui.panel.config.area_registry.picker.header")}
           </span>
           <span slot="introduction">
-            Areas are used to organize where devices are. This information will
-            be used throughout Home Assistant to help you in organizing your
-            interface, permissions and integrations with other systems.
+            ${this.hass.localize(
+              "ui.panel.config.area_registry.picker.introduction"
+            )}
             <p>
-              To place devices in an area, navigate to
-              <a href="/config/integrations">the integrations page</a> and then
-              click on a configured integration to get to the device cards.
+              <ha-markdown
```

Comment:
We should not run translations through markdown, that is a perfect place to inject malicious code !

Reply:
Already being done today: https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/config/js/automation.js

How otherwise are you going to workout blocks of text with links in them? Suggestion would be to rephrase and keep links out but then you can't have a friendly link just some icon maybe instead.

Reply:
Good catch, we should remove that one too.

Look, anyone can create a lokalise account and start creating stuff that we will render as markdown, including links to scam sites etc. This cannot be merged with translations being run through markdown.